packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -486,7 +498,17 @@ module.exports = class ImportEntries {
           return resolve();
         })
         .catch((error) => {
-          addlogs(this.config, chalk.red("Failed to create entries in '" + lang + "' language"), 'error');
+          addlogs(
+            this.config,
+            chalk.red(
+              "Failed to create entries in '" +
+                lang +
+                "' language. " +
+                'Title of the failed entry: ' +
+                JSON.parse(error.request.data).entry.title,
```

Comment:
"JSON.parse(error.request.data).entry.title" this would cause problems when any of the data received in unexpected structure. So we have to carefully check the existence of the properties here such as if(error?.request?.data) {. JSON.parse(error.request.data) }. or let title; try { title =JSON.parse(error.request.data).entry.title  } catch(){ ignore error } log()

Reply:
Made the change @shafeeqd959